test/e2e/installplan_e2e_test.go

```diff
@@ -2998,6 +3000,171 @@ var _ = Describe("Install Plan", func() {
 		ctx.Ctx().Logf(fmt.Sprintf("Install plan %s fetched with conditions %+v", fetchedInstallPlan.GetName(), fetchedInstallPlan.Status.Conditions))
 	})
 
+	When("waiting on the bundle unpacking job", func() {
+		var (
+			ns         *corev1.Namespace
+			catsrcName string
+			crc        versioned.Interface
+			ip         *operatorsv1alpha1.InstallPlan
+		)
+		BeforeEach(func() {
+			crc = newCRClient()
+			ns = &corev1.Namespace{}
+			ns.SetName(genName("ns-"))
+			Expect(ctx.Ctx().Client().Create(context.Background(), ns)).To(Succeed())
+
+			// Create the single (kiali) bundle catalog source
+			catsrc := &operatorsv1alpha1.CatalogSource{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      genName("kiali-"),
+					Namespace: ns.GetName(),
+					Labels:    map[string]string{"olm.catalogSource": "kaili-catalog"},
+				},
+				Spec: operatorsv1alpha1.CatalogSourceSpec{
+					Image:      "quay.io/olmtest/single-bundle-index:1.0.0",
```

Comment:
Does this need to be a real image ref? It looks like the unpacker only requires that a CatalogSource exists.

Reply:
You're right. Seems like we don't need a running CatalogSource here. Just a dummy one would do.